.travis.yml

```diff
@@ -0,0 +1,17 @@
+dist: trusty
+sudo: required
+language: c
+compiler:
+  - gcc
+  - clang
+
+script:
+  - make NANOPB_DIR=nanopb all
+  - make NANOPB_DIR=nanopb test
```

Comment:
Yeah, i need to actually fix the inclusion of googletest and didn't get around to that . It's on my TODO list along with getting the google3 code imported.